azure_monitor/tests/__init__.py

```diff
@@ -1,2 +1,2 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License.
\ No newline at end of file
+# Licensed under the MIT License.
```

Comment:
I thought the code in this repo was using Apache license

Reply:
Any repo under a `Microsoft` org can use the MIT license as long as it was created by a Microsoft employee. This was discussed with a CELA rep.

Reply:
I noticed we have multiple license headers in different files, do we want to use a single one or do we need to have all of them?

Reply:
We should be using MIT license consistently. Any other license used would be a bug.